docs/installation/slack/cloud-slack.mdx

```diff
@@ -8,21 +8,17 @@ The Botkube Cloud Slack App uses Botkube Cloud services to manage channels and r
 
 ## Prerequisites
 
-- A Botkube Cloud account with active subscription.
+- A Botkube Cloud account.
 
-  You can try out the Botkube Cloud Slack App for free by creating an account in the [Botkube Cloud app](https://app.botkube.io) and starting a free trial.
-
-:::note
-Once you downgrade your subscription to free, you will not be able to execute Botkube commands. However, you will still receive notifications from your clusters.
-:::
+  You can try out the Botkube Cloud Slack App for free by creating an account in the [Botkube Cloud app](https://app.botkube.io).
 
 ## Create a Botkube Cloud Instance with Cloud Slack
 
 1. Go to Botkube Cloud [Web App](https://app.botkube.io/) and click on `New Instance` button.
 
    ![New Instance](assets/cloud_slack_new_instance.png "Create new instance")
 
-2. Deploy Botkube into your environment.
+2. Install Botkube Cloud Agent on your Kubernetes cluster by following the instructions on the page.
```

Comment:
```suggestion
2. Install Botkube Agent on your Kubernetes cluster by following the instructions on the page.
```

do we need to introduce a new kind?  "Botkube Agent" and "Botkube Cloud Agent"? I would stay with Botkube Agent